platform/darwin/src/MGLShape.h

```diff
@@ -15,6 +20,14 @@ NS_ASSUME_NONNULL_BEGIN
 /** The subtitle of the shape annotation. The default value of this property is `nil`. */
 @property (nonatomic, copy, nullable) NSString *subtitle;
 
+#if TARGET_OS_IPHONE
+
+@property (nonatomic) BOOL canShowCallout;
+
+@property (nonatomic, nullable) UIView <MGLCalloutView> *calloutView;
```

Comment:
Fair enough, this is already a fairly ambitiously scoped pull request. I moved these properties to `MGLPointAnnotation` in 260ed7e.